### PR TITLE
Automation for manual testcases

### PR DESCRIPTION
**Pull request by abhimutant**
Automating the manual testcases for vm-create resource with Cloud-Init and NGT creds.

### CODE REVIEW

**Comment by siddharth-nutanix** [comment]
/ok-to-test
>Acceptance test run status: failure
>

File: nutanix/resource_nutanix_virtual_machine_test.go [hunk 1]
```diff
@@ -1121,3 +1192,108 @@ func testAccNutanixVMConfigResizeDiskClone(imgName, vmName string, diskSize int)
 		}
 	`, imgName, vmName, diskSize)
 }
+
+func testAccNutanixVirtualMachineCloudInitUserData(r int) string {
```
Comment (shreevari) [inline nutanix/resource_nutanix_virtual_machine_test.go]: All other config functions are in format`testAccNutanixVMConfig*`
Could we change this func to that format?

**Review by shreevari — APPROVED** [review]

File: nutanix/resource_nutanix_virtual_machine_test.go (hunk 1 above)
Comment (abhimutant) [reply nutanix/resource_nutanix_virtual_machine_test.go]: Sure. will do it.

**Comment by abhimutant** [comment]
/ok-to-test
>Acceptance test run status: failure
>

**Comment by siddharth-nutanix** [comment]
/ok-to-test
>Acceptance test run status: failure
>

**Comment by siddharth-nutanix** [comment]
/ok-to-test
>Acceptance test run status: failure
>

**Comment by abhimutant** [comment]
/ok-to-test

>Acceptance test run status: failure
>

**Comment by siddharth-nutanix** [comment]
/ok-to-test
>Acceptance test run status: success
>Line code coverage is 60.6

**Review by siddharth-kulshrestha — APPROVED** [review]

**Pull request merged**